### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ISaumya1011**
Please check the changes I've made according to your requirements  now. If any further changes need to be done, please let me know. N also, If you merge these changes, kindly accept my commits under hacktoberfest2023. Also, if possible please see if you can accept these changes against #28 and #30 both as it is necessary for my task completion under hacktoberfest2023. Thank you!

## Issue

Relevant [issue number](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Resolves #

## Testing Steps

1.

## Additional Information

1.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,29 +6,35 @@
 
 # Monster Hunter Now Events
 
+## Table Of Contents &#128203;
```
Comment (seriouslysean) [inline docs/README.md:9]: Remove this title entirely, it duplicates the goal of the main heading.

```suggestion
```

File: docs/README.md [hunk 2]
```diff
@@ -6,29 +6,35 @@
...
+* Calendar Subscription (Purpose)
+  - Subscribe to Calendar <br>
+* Local Implementation of Calendar Subscription <br>
+* Tools <br>
+* References 
+
+## Calendar Subscription &#128198;
```
Comment (seriouslysean) [inline docs/README.md:16]:
```suggestion
## Calendar Subscription
```

Remove the encoded entity.

File: docs/README.md [hunk 3]
```diff
@@ -6,29 +6,35 @@
...
 This is a calendar subscription file for events in [Monster Hunter Now](https://monsterhunternow.com/).
 
-If you aren't playing yet, [feel free to join](https://apps.apple.com/us/app/monster-hunter-now/id6445906110) and use my referral code for free items and zenny, `89K5MKCF`.
+In case you are not playing yet, you can join and play the game with me and my friends. [Feel free to join ](https://apps.apple.com/us/app/monster-hunter-now/id6445906110) and use my **referral code** for free items and zenny, *`89K5MKCF`*
```
Comment (seriouslysean) [inline docs/README.md:19]: This changes the tone be much more formal, leave it informal and just fix any spelling mistakes. You added bold tags around a code block, remove those.

File: docs/README.md [hunk 4]
```diff
@@ -6,29 +6,35 @@
...
-I created this because I didn't want to add events to a shard calendar for my friends and I any longer and it's also a great way to test out how beneficial AI is in coding via dialoge with ChatGPT, and an excuse to utilize the OpenAI API.
+I created this because I didn't want to add events to a shared calendar for me and my friends any longer. It's also a great way to visualize how AI is beneficial for coding via dialogue with ChatGPT, and an excuse to utilize the OpenAI API. 
 
-## Subscribe to Calendar
+## Subscribe to Calendar &#128204;
 
 Add the [ICS file](https://seriouslysean.github.io/monster-hunter-now-events/dist/events.ics) to your calendar app.
 
-```
+'''
```
Comment (seriouslysean) [inline docs/README.md:27]: You changed the three backticks denoting a code block to 3 single quotes which do not serve the same purpose, switch those back, please.

File: docs/README.md [hunk 5]
```diff
@@ -6,29 +6,35 @@
...
 https://seriouslysean.github.io/monster-hunter-now-events/dist/events.ics
-```
+'''
 
-## Run Locally
+## Local Implementation of Calendar Subscription
```
Comment (seriouslysean) [inline docs/README.md:31]: Leave this as "Run Locally". This change is too verbose.

File: docs/README.md [hunk 6]
```diff
@@ -6,29 +6,35 @@
...
-## Run Locally
+## Local Implementation of Calendar Subscription 
 
-### Open AI Key
-
-1. Sign up for an OpenAI API key, https://platform.openai.com/account/api-keys
-2. Add tokens if needed, https://platform.openai.com/account/billing/overview
-
-### Generate ICS File
+### * Open AI Key
```
Comment (seriouslysean) [inline docs/README.md:33]: These titles now start with an asterisk but shouldn't, remove those please.

File: docs/README.md [hunk 7]
```diff
@@ -45,17 +51,17 @@ This will do three things:
 3. `npm run generate:events-ics`
   - Generate the `dist/events.ics` file
 
-### Tools
```
Comment (seriouslysean) [inline docs/README.md:48]: This should still be an h3 as they're developer centric tools which fall under run locally.

File: docs/README.md [hunk 8]
```diff
@@ -45,17 +51,17 @@ This will do three things:
...
+## Tools 
 
-Download, Parse and Save One Article
-- `npm run fetch:article -- -u <url>`
+* Download, Parse and Save One Article
```
Comment (seriouslysean) [inline docs/README.md:56]: This is strictly a formatting change and doesn't change anything about the visual display.

File: docs/README.md [hunk 9]
```diff
@@ -45,17 +51,17 @@ This will do three things:
...
 
-Download, Parse and Save ALL New Articles
-- `npm run fetch:all-articles`
+* Download, Parse and Save ALL New Articles
+  - `npm run fetch:all-articles`
 
 Article `index.html` and `events.json` are output to `fixtures/{date}_news-{slug}`.
 
-## References
+## References &#128187;
```
Comment (seriouslysean) [inline docs/README.md:64]: Remove the entity.

**Review by seriouslysean — CHANGES_REQUESTED** [review]
The tone of the message is still being changed in a way I don't really like, and the overall organization doesn't match expectations.

I've left some more feedback if you're interested in continuing to refine it, though. I think a better understanding of what the project does might help, so it's a bit more clear where the elements belong.

If you're mostly interested in having this merged for hacktoberfest, it might be easier to start with just the spelling mistakes to resolve #28.

Thanks again for the time you've spent on this!

**Comment by seriouslysean** [comment]
Copying this here from #39 as it’s a helpful bit of context.

> Based on the naming of the branches, you’re making changes directly via GitHub. It might be worth learning to pull the changes down locally and using an editor instead.
> 
> https://code.visualstudio.com/docs/sourcecontrol/intro-to-git
> 
> Hope this helps!

**Comment by ISaumya1011** [comment]
Okay, thank you for the feedback! Actually my exams are approaching and I
will not be able to devote too much time on the project. Please allocate me
#28 . I'll complete it by EOD and provide the changes for the same. Will
look forward to work with you in the future!

**Comment by seriouslysean** [comment]
Closed in favor of https://github.com/seriouslysean/monster-hunter-now-events/pull/50.

**Pull request closed without merging**